src/__experimental__/components/textbox/__internal__/prefix.style.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+const StyledPrefix = styled.span`
+  align-self: center;
+  font-weight: 900;
+  margin-right: 4px;
```

Comment:
Spec says this gap should be 8px, should this be changed?

Reply:
input already has it's own padding, so it should add up to 8px gap